justfile

```diff
@@ -29,7 +29,7 @@ flash-secure-new BOARD=default_board +ARGS="":
 flash-frontier-new BOARD="frontier" +ARGS="":
     espflash write-bin --chip esp32c3 --port /dev/ttyACM0 --baud 921600 --no-stub 0x0 device/bootloader-frontier.bin {{ARGS}}
     espflash write-bin --chip esp32c3 --port /dev/ttyACM0 --baud 921600 --no-stub 0xD000 device/partitions.bin {{ARGS}}
-    just flash-secure {{BOARD}}
```

Comment:
Noting that removing this removes the `blank-otadata.bin`.

I wondered if [this scenario](https://github.com/frostsnap/frostsnap/pull/273#issuecomment-3218383227) could become relevant again:

1) Testing on non-secure boot device, do a firmware upgrade --> increments to `ota_1`.
2) Decide to `just flash-frontier-new`, enables firmware signature verification.
3) On boot, bootloader checks `ota_1`, signature not present, tries `ota_0` which would work(?).

So I think it's not a potential issue with this bootloader? But then this begs the question, why was I ever having issues with this when using the development bootloader in the first place?